tests/integration/household_composition/test_repeating_relationship.py

```diff
@@ -0,0 +1,37 @@
+from tests.integration.create_token import create_token
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestRepeatingRelationship(IntegrationTestCase):
+
+    def setUp(self):
+        super(TestRepeatingRelationship, self).setUp()
+
+        self.token = create_token('household', 'census')
+        resp = self.client.get('/session?token=' + self.token.decode(), follow_redirects=False)
+        is_home = {'permanent-or-family-home-answer': 'Yes'}
+        resp = self.client.post(resp.location, data=is_home, follow_redirects=False)
+        self.household_composition_location = resp.location
+
+    def test_should_ask_twenty_four_relationships_when_twenty_five_max_limit_and_twenty_six_people_added(self):
```

Comment:
Perhaps remove the numbers here and capture it in the when and then, otherwise it is a real mouth full and hard to follow. something like test_relationship_count_when_max_people_reached